calcom/cal.com
# fix: [CAL - 1831] resizing of colorpicker.

## What does this PR do?

This PR fixes the resizing issue of colorpicker when the color is updated.



Fixes #9228
Fixes #9263

/claim #9263


https://github.com/calcom/cal.com/assets/49358949/28540b30-d5c4-4595-b742-02d097c9167d

**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change 



which fixes an issue)

## How should this be tested?
Go to settings > Appearance. Try updating brand color using colorpicker. You won't see the width shrinking down now.



## Checklist
- [x] Fixed the colorpicker auto resize issue
